src/modules.json

```diff
@@ -395,7 +395,7 @@
                        "modules/iotjs_module_websocket.c"],
       "init": "iotjs_init_websocket",
       "js_file": "js/websocket.js",
-      "require": ["crypto", "events", "net", "util"]
+      "require": ["crypto", "events", "net", "util", "tls"]
```

Comment:
If I remember correctly the websocket can be used over simple net connections also ("ws://") and for that there is no tls requirement. Did you have a sample where the unsecure websocket connection was not working without the tls module? Of course having using websocket with tls should be the norm but having a hard requirement on tls (that is: the tls could not be turned off) would increase the binary size if the tls in not required in a target application.

Reply:
`websocket.js` always runs to fail because it require `tls` at start of source and `tls` native module not found.  I agree it's better to let `websocket` require `tls` only in `wss` scheme, we should change `websocket.js`.

Reply:
Ohh... I've misunderstood somewhere. I've just checked the websocket without tls and there is indeed a bit of problem. What we can do is add a `try-catch` when the tls module is required (just like in the module.js) and also fix the tls usages in the websocket.js. Could you update the PR or should I create one with the fix?

Reply:
OK I re-made the two commit